CHANGELOG.md

```diff
@@ -3,7 +3,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/)
 
-## 2.2.0 - 2022-5-16
+## 3.0.0 - 2022-10-07
+
+### Updated
+
+Relax dependency to `rubocop` to enable support for Ruby 3.
+
+Rubocop breaking changes:
```

Comment:
I would link rubocop's [changelog](https://github.com/rubocop/rubocop/blob/v0.92.0/CHANGELOG.md) here to know where these came from.

Reply:
I'd agree